llama_stack/distribution/configure.py

```diff
@@ -62,7 +62,7 @@ def configure_api_providers(config: StackRunConfig, build_spec: DistributionSpec
     if config.apis:
         apis_to_serve = config.apis
     else:
-        apis_to_serve = [a.value for a in Api if a not in (Api.telemetry, Api.inspect)]
+        apis_to_serve = [a.value for a in Api if a not in (Api.telemetry, Api.inspect, Api.providers)]
```

Comment:
whats the rationale behind adding these apis to the no-serve list in one place but to-be-served in other places in this branch? i know its following the same pattern as inspect so i guess my question applies to all such apis

Reply:
providers is not a routable API, so in some places it needs to be skipped else we will check the routing table and will error out, that's why in some places we manually add it in like we do with inspect